### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by GochoMugo**
Feature:

* Basic tests have been added
* Ready to be integrated with Travis CI

References:

* Call for tests: https://github.com/kamikazechaser/tg-resolve/issues/4

### CODE REVIEW

**Comment by kamikazechaser** [comment]
Thanks for this

File: test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+// Node Packages
+
+var should = require('should');
+var tgresolve = require('.');
+
+// Module variables
+
+var username = "@kamikazechaser";
+var userid = 135207785;
+var token = process.env.TELEGRAM_TOKEN;
```
Comment (GochoMugo) [inline test.js:18]: Just remember we need a token saved in the CI.

Comment (kamikazechaser) [reply test.js:18]: Test Passed

Comment (GochoMugo) [reply test.js:18]: Yeah! Feels great!

**Pull request merged**